app/experimenter/kinto/diagrams/diagrams.sequence

```diff
@@ -76,46 +78,52 @@ end
 note over Experimenter Backend: The scheduled background task is invoked\nand finds the change rejected in the\ncollection
 group #00ffff Worker Updates Experiment
 Experimenter Worker->Remote Settings Backend:Check collection after timeout
-Experimenter Worker->Experimenter Backend:<align:center>Experiment\n(draft/idle)\n+changelog
+Experimenter Worker->Experimenter Backend:<align:center>Experiment\ndraft/idle(none)\n+changelog
 end
 
 
 # Publish New Experiment (approve/timeout)
 note over Experiment Reviewer:The experiment owner is ready\nto launch their experiment and\nclicks the launch button
 group #ff00ff Owner Launches in Experimenter
 Experiment Owner->Experimenter UI:Launch Experiment
-Experimenter UI->Experimenter Backend:<align:center>Experiment\n(draft/review)\n+changelog
+Experimenter UI->Experimenter Backend:<align:center>Experiment\ndraft/review(live)\n+changelog
 end
 note over Experiment Reviewer:The experiment reviewer reviews the\nexperiment's details on the summary\npage and clicks the approve button
 group #ffff00 Reviewer Approves in Experimenter
 Experiment Reviewer->Experimenter UI:Approve
-Experimenter UI->Experimenter Backend:<align:center>Experiment\n(draft/approved)\n+changelog
+Experimenter UI->Experimenter Backend:<align:center>Experiment\ndraft/approved(live)\n+changelog
 end
 note over Experimenter Backend:The scheduled background task is\ninvoked, finds an approved experiment\nto launch, and creates a new record\nwith the serialized DTO
 group #00ffff Worker Publishes to Remote Settings
-Experimenter Backend->Experimenter Worker:<align:center>Find Experiments:\ndraft/approved
+Experimenter Backend->Experimenter Worker:<align:center>Find Experiments:\ndraft/approved(live)
 Experimenter Worker->Remote Settings Backend:<align:center>Create Record\nStatus: to-review
-Experimenter Worker->Experimenter Backend:<align:center>Experiment\n(draft/waiting)
+Experimenter Worker->Experimenter Backend:<align:center>Experiment\ndraft/waiting(live)
 end
 note over Experimenter Backend:The scheduled background task is\ninvoked, finds a pending unattended\nreview, rolls it back, and reverts the\nexperiment back to the review state
 group #00ffff Worker Rolls Back Remote Settings
 Experimenter Worker->Remote Settings Backend:Check collection after timeout
 Experimenter Worker->Remote Settings Backend:<align:center>Rollback\nStatus: to-rollback
-Experimenter Worker->Experimenter Backend:<align:center>Experiment\n(draft/review)\n+changelog
+Experimenter Worker->Experimenter Backend:<align:center>Experiment\ndraft/review(live)\n+changelog
 end
 
+### Ending Enrollment
 
-# Update Live Experiment (task/approve)
```

Comment:
Yeah so, yes we will totally have different kinds of live updates when we add the ability to like change population sizes or targeting or whatever while an experiment is live, but I *think* we'll have to possibly change some of the machinery when that's the case, so since we only support ending enrollment right now I thought it was best for the diagrams to describe specifically the cases that are supported now.  

And the `is_paused` bool is kind of incidental to this workflow I think, I wanted to just describe the state transition/interactions with rs, the actual value of is_paused isn't used in any of the filtering or state transitions so I didn't think it needed to be included.  Maybe if it turns out it needs to be we can update them.

Reply:
> And the is_paused bool is kind of incidental to this workflow I think, I wanted to just describe the state transition/interactions with rs, the actual value of is_paused isn't used in any of the filtering or state transitions so I didn't think it needed to be included.

@jaredlockhart [`is_paused` *is* actually used in the filters](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/models/nimbus.py#L148-L159), which is why I mentioned it

Reply:
Oh right now yes, I mean after we change it to use status/publish_status/status_next.  In theory that should be sufficient to do the whole workflow.  Like you set the experiment to live/review/live and wahtever's changed in teh experiment will get reserialized and repushed, that just happens to be is_paused right now.

Reply:
Yeah, that was basically my comment: This diagram is specifically for pausing, so I thought it should mention is_pause. Or change it to be generic to all updates rather than just pause.

Reply:
No I get you, let's see how it shakes out when we implement it and if mentioning is_paused is critical to the workflow then we can add it 👍